cypress/support/server.js

```diff
@@ -1250,3 +1255,91 @@ export const mockPostAvatar = (shouldThrowError) => {
     },
   ).as('uploadAvatar');
 };
+
+export const mockGetCategoryTypes = (categoryTypes) => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(
+        `${API_HOST}/${parseStringToRegExp(GET_CATEGORY_TYPES_ROUTE)}$`,
+      ),
+    },
+    ({ reply }) => {
+      reply(categoryTypes);
+    },
+  ).as('getCategoryTypes');
+};
+
+export const mockGetCategories = (categories, shouldThrowError) => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(
+        `${API_HOST}/${parseStringToRegExp(buildGetCategoriesRoute())}`,
+      ),
+    },
+    ({ reply }) => {
+      if (shouldThrowError) {
+        reply({ statusCode: StatusCodes.BAD_REQUEST, body: null });
+        return;
+      }
+      reply(categories);
+    },
+  ).as('getCategories');
+};
+
+export const mockGetItemCategories = (items, shouldThrowError) => {
+  cy.intercept(
+    {
+      method: DEFAULT_GET.method,
+      url: new RegExp(
+        `${API_HOST}/${parseStringToRegExp(
+          buildGetItemCategoriesRoute(items[0]?.id),
```

Comment:
Probably we don't want to only mock for the first item. I know it's difficult to use, but could you use the ID_FORMAT? Or hardcode the endpoint can be a temporary solution. It's probable that the regex is not well parsed (because of `-`?)